internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -18594,7 +18594,6 @@
         "dc/1fxw60rxykwz1",
         "dc/1fy9mh75bmh6b",
         "dc/1fyly2lmyfgk6",
-        "dc/1fz8xftk06k68",
```

Comment:
do we expect these dropped stat vars?

Reply:
Good eye.  No, not expected.

I did a binary search over BQ for these SVs, and it feels like this might be from updating the SuperEnum aggregations in cl/441569548.  @Spaceenter - could you help look further?

Build on April 12th finds these SVs in 3 SuperEnum imports: https://screenshot.googleplex.com/4PBCh89cwjSy7Er
Build on April 13th doesn't find them: https://screenshot.googleplex.com/3njmmgHuhC4ACpu

Reply:
Actually, this is an improvement.  These three "auto-gen" SVs are equivalent to curated SVs 'Count_Person_55To64Years', 'Count_Person_55To64Years_Female', 'Count_Person_55To64Years_Male'.

Turns out that with the new change, the "SuperEnum" imports attach the stats to the "curated" SVs, rather than the "auto-gen" SVs.

For proof, if we look at the April 12th build, the curated SVs don't have stats from "*SuperEnum" imports: https://screenshot.googleplex.com/BsbiE3fztHJvCKX

But if we look at the April 13th build (post cl/441569548), they do: https://screenshot.googleplex.com/3cCthAgucf9qtRv

So, yay!

Reply:
thanks for the investigation! yay!